avocado/utils/kernel.py

```diff
@@ -120,8 +126,17 @@ def configure(self):
             build.make(self.linux_dir, extra_args='-C %s olddefconfig' %
                        self.linux_dir)
         else:
-            build.make(self.linux_dir, extra_args='-C %s defconfig' %
-                       self.linux_dir)
+            build.make(self.linux_dir, extra_args='-C %s %s' % (self.linux_dir,
+                       " ".join(targets)))
+        if extra_configs:
+            config_file = tempfile.NamedTemporaryFile(mode='w+t',
```

Comment:
This file will eventually be deleted once it's garbage collected, but it'd be better to make it explicit and use it as context manager.

Reply:
Indeed I initially implemented this using context manager but then I switch to this, don't remember why. OK, going to change on v2.